lte/gateway/python/integ_tests/Makefile

```diff
@@ -22,17 +22,18 @@ $(PYTHON_BUILD)/setupinteg_env:
 	@echo export PYTHONPATH=$(PYTHONPATH):$(S1AP_TESTER_PYTHON_PATH) >> $(PYTHON_BUILD)/bin/activate
 	touch $(PYTHON_BUILD)/setupinteg_env
 
-# TODO T21489739 - Don't sleep and don't stop after a failure
 RESULTS_DIR := /var/tmp/test_results
 ifdef enable-flaky-retry
 	FLAKY_CMD_ARGS := --force-flaky --no-flaky-report --max-runs=3 --min-passes=1
 else
 	FLAKY_CMD_ARGS :=
 endif
+# Don't remove the ending comment from function execute_test. Somehow without
+# any ending statement the failing testcases are missing status update
 define execute_test
 	echo "Running test: $(1)"
 	timeout --foreground -k 930s 900s sudo -E PATH=$(PATH) PYTHONPATH=$(PYTHONPATH):$(S1AP_TESTER_PYTHON_PATH) $(BIN)/pytest $(FLAKY_CMD_ARGS) --capture=tee-sys --junit-xml=$(RESULTS_DIR)/$(basename $(notdir $(1))).xml -x $(1) || (echo "fail" > $(MAGMA_ROOT)/test_status.txt && exit 1)
-	sleep 1
+	# Testcase $(1) execution completed
```

Comment:
Is this comment needed here?

Reply:
This comment is not mandatory, but when exiting for the failing the test cases in the previous statement, if there is no following statement after that, somehow the failure status is not updated to the terminal. This is the reason why there was a sleep added there earlier. I have replaced the sleep with the comment to reduce the execution time. Moreover I have mentioned this point in line 31 above for clarity when working with this file.